stock_change_qty_reason/wizard/stock_change_product_qty.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2016-2017 ACSONE SA/NV (<http://acsone.eu>)
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+"""stock_change_product_qty"""
+
+
+from odoo import models, fields, api
+
+
+class StockChangeProductQty(models.TransientModel):
+    """Class to inherit model stock.change.product.qty"""
+    _inherit = 'stock.change.product.qty'
+
+    reason = fields.Char('Reason',
+                         help='Type in a reason for the '
+                         'product quantity change')
+    encoded_reason = fields.Many2one('stock.change.product.reason',
```

Comment:
better use a model such as 'stock.inventory.qty.change.reason'
and a field reason_id instead of reason

Reply:
comments attended @jbeficent